### ISSUE TITLE
Revert post privacy selection from modal to dropdown

### ISSUE THREAD

**Issue by wohali** [glitch-soc/mastodon#3341]
### Pitch

If upstream does not take this PR, please consider it for glitch-soc.

https://github.com/mastodon/mastodon/pull/37346

### Motivation

As per https://github.com/mastodon/mastodon/issues/36742 the changes in 4.5 to this dialog are a UX failing and an accessibility nightmare.
